shylands/climatechoice
# Accessibility tweaks and console error fixes

## Done

These changes fix up some small console / dev log errors and accessibility tweaks to improve the overall accessibility of the site.

Implementing these changes will give the site an almost perfect accessibility score on Lighthouse, except for one remaining issue - color contrasts across the site, which I've left as they require a design eye.

The biggest issue is light text on green backgrounds which is difficult for red/green color-blind users and light grey text on light backgrounds which is difficult for partially sighted users. 

I'ved add color contrasts as a [separate issue](https://github.com/shylands/climatechoice/issues/63).

## QA

- Pull feature branch
- Check site runs as expected
- Sanity check the code
